vignettes/manual/Constraints.Rmd

```diff
@@ -36,6 +36,9 @@ then the model's ocean carbon uptake (in Pg C) will follow this contents of this
 Ca_constrain=csv:path/to/constraint/file.csv
 ```
 
-then the model's atmospheric carbon pool (given in ppmv CO2) will follow this contents of this file (which should have two columns, `Date` and `Ca_constrain`). The model will extrapolate within this data as necessary, but not beyond its endpoints. If the constraint [CO2] is lower than the calculated [CO2] at that time step, any excess is moved to the deep ocean; conversely, any necessary deficit is transferred from there.
+then the model's atmospheric carbon pool (given in ppmv CO2) will follow this contents of this file (which should have two columns, `Date` and `Ca_constrain`). If the constraint [CO2] is lower than the calculated [CO2] at that time step, any excess is moved to the deep ocean; conversely, any necessary deficit is transferred from there.
```

Comment:
This is not correct.  The atmospheric carbon pool is different from the atmospheric CO<sub>2</sub> concentration.  The latter is what we are actually constraining.

Reply:
Good catch. Addressed in c841d8f.